src/status_im/wallet/prices.cljs

```diff
@@ -32,10 +32,14 @@
 
 (fx/defn on-update-prices-success
   {:events [::update-prices-success]}
-  [{:keys [db]} prices]
-  {:db (assoc db
-              :prices prices
-              :prices-loading? false)})
+  [{:keys [db]} prices currency-code]
+  {:db (if (contains? prices currency-code)
```

Comment:
`prices` is a map with from keys, currency-code is a vector with to key, not sure how this contains? will work

Reply:
i guess we need to assign prices in any case. but iterate  prices map and if there is no to equal to key from `currency-code` code vector we also additionally should assoc an error